mapper/mapper.go

```diff
@@ -58,6 +60,48 @@ func MainFigure(ctx context.Context, id, datePeriod string, figure zebedee.Times
 	return &mf
 }
 
+func ReleaseCalendar(rawReleaseCalendar release_calendar.ReleaseCalendar) *model.ReleaseCalendar {
+	releaseResults := *rawReleaseCalendar.Result.Results
+	numReleasesScheduled := rawReleaseCalendar.Result.NumberOfResults
+
+	for i := len(releaseResults) - 1; i >= 0; i-- {
+		if releaseResults[i].Description.Cancelled || !releaseResults[i].Description.Published {
+			numReleasesScheduled--
+		}
+	}
+	rc := model.ReleaseCalendar{
+		Releases:         getLatestReleases(releaseResults),
+		NumberOfReleases: strconv.Itoa(numReleasesScheduled),
+	}
+	return &rc
+}
+
+func getLatestReleases(rawReleases []release_calendar.Results) []model.Release {
+	var latestReleases []model.Release
+
+	// Removed canceled releases or unpublished releases
+	for i := len(rawReleases) - 1; i >= 0; i-- {
+		if rawReleases[i].Description.Cancelled || !rawReleases[i].Description.Published {
+			rawReleases = append(rawReleases[:i], rawReleases[i+1:]...)
+		}
+	}
+
+	// Reverse order
+	sort.Slice(rawReleases, func(i, j int) bool {
+		return rawReleases[i].Description.ReleaseDate.After(rawReleases[j].Description.ReleaseDate)
+	})
+	for i := 0; i < 3; i++ {
```

Comment:
Consider defining a constant for the magic number `3` for clarity. (assuming that it is a constant?)